README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
 You can also [download the compiled and minified assets (CSS, JavaScript) from
 GitHub](docs/installation/installing-from-dist.md).
 
-## Browser support
+## Accessibility
```

Comment:
I'm not entirely sure about getting rid of the "Browser support" heading as browser support doesn't only relate to accessibility but is wider than that 🤔

Reply:
See my comment here - https://github.com/alphagov/govuk-frontend/pull/1616#discussion_r337024332. Would that proposed heading structure work better?

Reply:
That sounds good, thanks @m-green  👍 Non-blocking but I'd then prefer to see the link to the supported browsers under that main heading, rather than one level down under `### Accessibility`.